test/basic.test.js

```diff
@@ -377,3 +377,21 @@ test('normalize number to string with an object', function (t) {
 
   instance.info({ answer: 42 }, 1)
 })
+
+test('handles objects with null prototype', function (t) {
+  var instance = pino(sink(function (chunk, enc, cb) {
+    delete chunk.time
```

Comment:
What about a test for `undefined` values?
It's basically already covered in other tests, so not necessarily needed.